packages/editor/src/serialization/html/htmlSerializers.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2025-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import type { SlateSerializer } from "../../core";
+import { breakSerializer } from "../../plugins/break/breakSerializer";
+import { headingSerializer } from "../../plugins/heading/headingSerializer";
+import { linkSerializer } from "../../plugins/link/linkSerializer";
+import { listSerializer } from "../../plugins/list/listSerializer";
+import { markSerializer } from "../../plugins/mark/markSerializer";
+import { paragraphSerializer } from "../../plugins/paragraph/paragraphSerializer";
+import { sectionSerializer } from "../../plugins/section/sectionSerializer";
+
+// TODO: This is probably not the right place for this. Should we require consumers to put this together?
```

Comment:
Tror jeg syntes det gir mening ettersom det helt sikkert vil variere 🤔

Reply:
Altså at det gir mening å ikke ha dette her?

Reply:
Jepp stemmer, altså å tvinge consumers til å definere det, jeg syntes hvertfall det virker mest ryddig!

Reply:
Men vi tenker da fortsatt å eksportere våre serializers fra her right? Vi kan jo tilby en "ferdig" html serializer/deserializer som fungerer 😄

Reply:
Vi eksporterer serializers, konsumenter setter det sammen. Alle vil ha ulikt setup, så det er ikke stort poeng i å sette det sammen for de.